ansible/roles/test/files/ptftests/arista.py

```diff
@@ -148,7 +148,7 @@ def run(self):
             'interfaces']['Port-Channel1']['lastStatusChangeTimestamp']
         samples[cur_time] = sample
 
```

Comment:
May be add a TODO in comments to revert this change after FRR issue (w/ link) is fixed.

Reply:
Done.